src/browser/chrome.ts

```diff
@@ -157,8 +191,40 @@ export const openUrl = async ({
         msg.type() === "error" &&
         log(`JS error |`, msg.text().replace(/\n[\w\W]*/, ""), `@ [${url}]`)
     );
+
+    let counterRequestsUnfinished = 0;
+    const updateCounter = async (counter: number) => { 
+      counterRequestsUnfinished = counter;
+      try {
+        await page.evaluate(({ counterRequestsUnfinished, COUNTER_REQUEST_UNFINISHED_VARIABLE_NAME }) => {
+          const item = counterRequestsUnfinished.toString();
+          localStorage.setItem(COUNTER_REQUEST_UNFINISHED_VARIABLE_NAME, item);
+        }, { counterRequestsUnfinished, COUNTER_REQUEST_UNFINISHED_VARIABLE_NAME });
+      } catch (e) {}
```

Comment:
Silencing errors can make us debug it. Better to `console.error` if you want soft error handling